### DISCUSSION TITLE
Will we support the Golang version of ros2msg parser?

### DISCUSSION THREAD

**Discussion by StarCsu** [foxglove/mcap#1111] [General]
i'm a golang engineer,need to use go mcap repo recently,found that there is currently no ros2msg parser;

are you consider supporting ros2msg parser？

**Comment by defunctzombie** [comment]
We currently have no plans to write a ros2msg parser in golang; however we would review community contributions.

**Reply by StarCsu** [reply]
tks，i got it;if my company need parser，i will consider implementing it;
